src/threads/table.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { forumTable } from './table';
+
+describe('Forum table', () => {
+  const participantId = '12';
+  const itemId = '42';
+  const userId = '1001';
+
+  it('should add a thread event and retrieve new event in thread events', async () => {
+    await expect(forumTable.addThreadEvent(participantId, itemId, { type: 'thread_opened', byUserId: userId })).resolves.toBe(undefined);
```

Comment:
The `.resolves.toBe(undefined);` is just checking that the function does what its definition does ? This is just checking the TS compiler. 

We just have to check that the promise returns something and that this is not an error. Is `.resolves.toBe(undefined);`  the best way?

Reply:
Fixed

Reply:
Considering the body of the method, yes. The returned value is constructed from the input and returned if aws call resolves, I don't see much interest in testing that.